lib/puppet/file_bucket/file.rb

```diff
@@ -79,6 +78,10 @@ def self.from_s(contents)
     self.new(contents)
   end
 
+  def self.from_binary(contents)
+    self.new(contents)
+  end
+
```

Comment:
since we're removing `:s` as a supported format, I think we should remove the `from_s` and `to_s` methods as well, and rename the `to_s` method to `to_binary` for `StringContents` and `FileContents`